config/config.go

```diff
@@ -15,13 +15,14 @@
 package config
 
 // Config contains shared config parameters, common to the source and
-// destination. If you don't need shared parameters you can entirely remove this
-// file.
+// destination.
 type Config struct {
 	// Token is used to authenticate API access.
 	Token string `json:"token" validate:"required"`
 	// Url is the remote influxdb host for api calls.
 	URL string `json:"url" validate:"required"`
 	// Org is an organization name or ID.
 	Org string `json:"org" validate:"required"`
+	// Bucket is the database name to access.
```

Comment:
Bucket != database. Shouldn't this be `the bucket name to access` ?

Reply:
updated.